package.json

```diff
@@ -21,8 +21,8 @@
   },
   "homepage": "https://github.com/redbadger/immutable-cursor",
   "dependencies": {
-    "atom-store": "0.0.2",
-    "immutable": "3.7.6"
+    "atom-store": "0.0.3",
```

Comment:
We'll need to cut a new major version of `atom-store` due to https://github.com/jameshopkins/atom-store/commit/e809f638eb35566a38e72e4a5d8fb02630512ebd and potentially any other changes to the interface we want to rollup into the next release.